intl.emu

```diff
@@ -79,17 +79,13 @@
           1. Set _x_ to 0.
         1. <ins>If _x_ is a Decimal128 value, then</ins>
           1. <ins>Let _sign_ be *undefined*.</ins>
-          1. <ins>Assert: _x_ is not *NaN*<sub>𝔻</sub>.</ins>
-          1. <ins>Assert: _x_ is not *+∞*<sub>𝔻</sub>.</ins>
-          1. <ins>Assert: _x_ is not *-∞*<sub>𝔻</sub>.</ins>
-          1. <ins>Let _v_ be _x_[0].</ins>
+          1. <ins>Let _v_ be cohort(_x_).</ins>
```

Comment:
Issue: You should at some point here pull the precision from the Decimal to retain trailing zeros.

Reply:
(ditto)